printer_mods/evandepol/Crydom-GN-Guard/README.md

```diff
@@ -0,0 +1,12 @@
+# Crydom GN series terminal guard
+
+I replaced the SSR on my build with a Crydom GN series SSR. The model i received didn't have the terminal cover, so I create a simple cover that is secured with the mounting screw on the SSR, and helps prevent accidental touching of the high voltage terminals.
+
+![Model](Images/guard.png)
+
```

Comment:
The mounting hole on top prints in air, it needs a chamfer or some other built-in way to support it.

<img width="344" alt="Screen Shot 2021-05-19 at 08 06 09" src="https://user-images.githubusercontent.com/69389593/118810015-5c0f6b00-b879-11eb-90de-c2fc47d55b97.png">

Reply:
Fixed by using a bridging approach over 2 slicing layers. Performing a test print now to confirm but should be clean.

Reply:
![IMG_0898](https://user-images.githubusercontent.com/12777482/119205744-26c87000-ba67-11eb-9aec-1f87cf43d549.jpg)